runtime/compiler/runtime/J9Runtime.hpp

```diff
@@ -62,29 +62,29 @@ void replaceFirstTwoBytesWithData(void *startPC, int32_t startPCToData);
 
 #if defined(TR_HOST_POWER)
 #define  OFFSET_REVERT_INTP_PRESERVED_FSD                (-4)
-#define  OFFSET_REVERT_INTP_FIXED_PORTION                (-12-2*sizeof(intptr_t))
+#define  OFFSET_REVERT_INTP_FIXED_PORTION                (-(12 + 2 * (int32_t)sizeof(intptr_t)))
```

Comment:
I added the cast because of an offline conversation, the problem is that `sizeof` returns an unsigned integer of "at least 16 bits", meaning on some compilers `OFFSET_REVERT_INTP_FIXED_PORTION` may end up being unsigned. Since it's being passed into a function expecting `int32_t` it would be safer to ensure `OFFSET_REVERT_INTP_FIXED_PORTION` is signed. Am I right about that?

Reply:
i assumed these macros are used in the same context. it looks strange to me that `OFFSET_REVERT_INTP_FIXED_PORTION` needs the cast but `OFFSET_SAMPLING_BRANCH_FROM_STARTPC ` etc. don't.

Reply:
Since all of these macros are supposed to have negative values, I think it would be safest for all of them to have the cast.